eZ/Publish/Core/Repository/Repository.php

```diff
@@ -879,6 +744,28 @@ public function getFieldTypeService()
     }
 
     /**
+     * Get PermissionService.
+     *
+     * @return \eZ\Publish\API\Repository\PermissionService
+     */
+    public function getPermissionService()
+    {
+        if ($this->permissionService === null) {
+            $this->permissionService = new PermissionService(
+                $this,
+                $this->getUserService(),
```

Comment:
Removed in https://github.com/ezsystems/ezpublish-kernel/pull/1720/commits/53be41ef3fb217330a7127d15551c26989f711d2.